### PR TITLE
run through grammarly

### LINKED ISSUE #134: Run our *.md documents in this repo through the Grammer.ly service and update
#### Issue Summary

We would like to get our English polished up throughout the repo.

#### Aceptance Criteria

* Every .md file in this repo has been run through the Grammer.ly service and updated accordingly

### PR DESCRIPTION
**Pull request by tyoungNIO**
Resolves #134
Only a few changes were needed.

# Fixes # 

### Checklist
- [x] I have made a material change to the repo (functionality, testing, spelling, grammar)
- [x] I have read the [Contribution Guide] and my PR follows them.
- [x] I updated my branch with the master branch.
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation about the functionality in the appropriate .md file
- [x] I have added in line documentation to the code I modified

### Short description of what this PR does:
- 
- 

If you have questions, please send an email to [Sendgrid](mailto:dx@sendgrid.com), or file a Github Issue in this repository.

### CODE REVIEW

**Comment by SendGridDX** [comment]
[CLA assistant check](https://cla.sendgrid.com/sendgrid/open-source-library-data-collector?pullRequest=135) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.sendgrid.com/sendgrid/open-source-library-data-collector?pullRequest=135) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.sendgrid.com/check/sendgrid/open-source-library-data-collector?pullRequest=135) it.</sub>

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # SendGrid Community Code of Conduct
 
-The SendGrid open source community is made up of members from around the globe with a diverse set of skills, personalities, and experiences. It is through these differences that our community experiences successes and continued growth. When you're working with members of the community, we encourage you to follow these guidelines, which help steer our interactions and strive to maintain a positive, successful and growing community. 
+The SendGrid open source community is made up of members from around the globe with a diverse set of skills, personalities, and experiences. It is through these differences that our community experiences successes and continued growth. When you're working with members of the community, we encourage you to follow these guidelines, which help steer our interactions and strive to maintain a positive, thriving and growing community.
```
Comment (hugovk) [inline CODE_OF_CONDUCT.md]: I think this is less clear, it changes the meaning a bit.

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -9,15 +9,15 @@ Members of the community are open to collaboration, whether it's on pull request
...
 
-## Additional Guidance 
+## Additional Guidance
 
 ### Disclose Potential Conflicts of Interest
-Community discussions often involve interested parties. We expect participants to be aware when they are conflicted due to employment or other projects they are involved in and disclose those interests to other project members. When in doubt, over-disclose. Perceived conflicts of interest are important to address so that the community’s decisions are credible even when unpopular, difficult or favorable to the interests of one group over another.
+Community discussions often involve interested parties. We expect participants to be aware when they are conflicted due to employment or other projects they are involved in and disclose those interests to other project members. When in doubt, over-disclose. Perceived conflicts of interest are essential to address so that the community’s decisions are credible even when unpopular, difficult or favorable to the interests of one group over another.
```
Comment (hugovk) [inline CODE_OF_CONDUCT.md]: I think this is less clear.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
-Hello! Thank you for choosing to help contribute to one of the SendGrid open source projects. There are many ways you can contribute and help is always welcome.  We simply ask that you follow the following contribution policies.
+Hello! Thank you for choosing to help contribute to one of the SendGrid open source projects. There are many ways you can contribute and help is always welcome.  We merely ask that you follow the following contribution policies.
```
Comment (hugovk) [inline CONTRIBUTING.md]: I think this is less clear.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -11,14 +11,14 @@ Hello! Thank you for choosing to help contribute to one of the SendGrid open sou
 - [Creating a Pull Request](#creating-a-pull-request)
 
 <a name="roadmap"></a>
-We use [Milestones](https://github.com/sendgrid/open-source-library-data-collector/milestones) to help define current roadmaps, please feel free to grab an issue from the current milestone. Please indicate that you have begun work on it to avoid collisions. Once a PR is made, community review, comments, suggestions and additional PRs are welcomed and encouraged.
+We use [Milestones](https://github.com/sendgrid/open-source-library-data-collector/milestones) to help define current roadmaps, please feel free to grab an issue from the current milestone. Please indicate that you have begun work on it to avoid collisions. Once a PR is made, community review, comments, suggestions, and additional PRs are welcomed and encouraged.
```
Comment (hugovk) [inline CONTRIBUTING.md]: This is fine, the docs' style guide says to use Oxford commas.

https://github.com/sendgrid/docs/blob/develop/styleguide.md#punctuation

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -33,20 +33,20 @@ If you'd like to make a feature request, please read this section.
 
 The GitHub issue tracker is the preferred channel for library feature requests, but please respect the following restrictions:
 
-- Please **search for existing issues** in order to ensure we don't have duplicate bugs/feature requests.
+- Please **search for existing issues** to ensure we don't have duplicate bugs/feature requests.
```
Comment (hugovk) [inline CONTRIBUTING.md]: 👍

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -33,20 +33,20 @@ If you'd like to make a feature request, please read this section.
...
 - Please be respectful and considerate of others when commenting on issues
 
 <a name="submit-a-bug-report"></a>
 ## Submit a Bug Report
 
 Note: DO NOT include your credentials in ANY code examples, descriptions, or media you make public.
 
-A software bug is a demonstrable issue in the code base. In order for us to diagnose the issue and respond as quickly as possible, please add as much detail as possible into your bug report.
+A software bug is a demonstrable issue in the code base. For us to diagnose the issue and respond as quickly as possible, please add as much detail as possible into your bug report.
```
Comment (hugovk) [inline CONTRIBUTING.md]: 👍

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -149,7 +149,7 @@ All PRs require passing tests before the PR will be reviewed.
 
 All test files are in the [`test`](https://github.com/sendgrid/open-source-library-data-collector/tree/master/test) directory.
 
-For the purposes of contributing to this repo, please update the [`test.py`](https://github.com/sendgrid/open-source-library-data-collector/blob/master/test/test.py) file with unit tests as you modify the code.
+To contribute to this repo, please update the [`test.py`](https://github.com/sendgrid/open-source-library-data-collector/blob/master/test/test.py) file with unit tests as you modify the code.
```
Comment (hugovk) [inline CONTRIBUTING.md]: 👍

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -222,7 +222,7 @@ Please run your code through:
 
 4. Commit your changes in logical chunks. Please adhere to these [git commit
    message guidelines](http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html)
-   or your code is unlikely be merged into the main project. Use Git's
+   or your code is unlikely to be merged into the main project. Use Git's
```
Comment (hugovk) [inline CONTRIBUTING.md]: 👍

**Comment by tyoungNIO** [comment]
@hugovk thank you for the feedback, please see the newest commit to address your comments. Thanks!

**Pull request closed without merging**